### PR TITLE
Export Color Mode of chakra-ui 

### PR DESCRIPTION
**Pull request by benits**
### @chakra-ui/color-mode
React component that adds support for light mode and dark mode using
`localStorage` and `matchMedia`.
## Proposed changes
In this PR color-mode components and/or hooks were added that were exiting in the chakra-ui but were not being exported in the rock-ui.
Some of these components are necessary to finalize the delivery of the client interface.
Components added:
- ColorMode
- ColorModeContext
- ColorModeOptions
- ColorModeProvider
- ColorModeProviderProps
- ColorModeScript
- DarkMode
- LightMode
- StorageManager
- cookieStorageManager
- localStorageManager
- storageKey
- useColorMode
- useColorModeValue
## Does this close any currently open issues?
- Export Color Mode of chakra-ui - https://taiga6.rockcontent.com/project/rs-frontend/task/52
## Pull request checklist
_Put an `x`  in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._
Please check if your PR fulfills the following requirements:
- [x] Lint and unit tests pass locally with my changes
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)
- [x] Any dependent changes have been merged and published in downstream modules
## Pull request type
 
 
What types of changes does your code introduce to Rock-ui?
_Put an `x`  in the boxes that apply_
- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation Update (if none of the other choices apply)
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Other (please describe): 
## Does this introduce a breaking change?
- [ ] Yes
- [x] No

## Other information
N/A

<sup> Created from VS Code using [CodeStream](https://codestream.com/?utm_source=cs&utm_medium=pr&utm_campaign=github*com)</sup>

### CODE REVIEW

File: src/components/color-mode/stories/color-mode.stories.tsx [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+  };
+  return (
+    <header>
+      <ColorModeProvider options={options}>
+        <Button
+          _focus={{ outline: 'none' }}
+          onClick={toggleColorMode}
+          rightIcon={<Icon />}
+          color={colorMode === 'light' ? 'gray.500' : 'blue.500'}
+          // bg={colorMode === 'light' ? 'whiteAlpha.500' : 'gray.500'}
```
Comment (fernandacanto) [inline src/components/color-mode/stories/color-mode.stories.tsx:47]: Acho que podemos remover esses comentários, ou eles são necessários?

Comment (benits) [reply src/components/color-mode/stories/color-mode.stories.tsx:47]: Boa haha, acabei deixando passar vou remover.

Comment (benits) [reply src/components/color-mode/stories/color-mode.stories.tsx]: done

File: src/components/color-mode/utils.tsx [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+import { screen } from '@testing-library/react';
+import { ColorModeOptions } from './index';
+import theme from '@chakra-ui/theme';
+
+export const DummyComponent = () => {
+  const { useColorMode } = require('./index');
+  const { colorMode, toggleColorMode } = useColorMode();
+
+  return (
+    <button type="button" onClick={toggleColorMode}>
```
Comment (fernandacanto) [inline src/components/color-mode/utils.tsx:11]: Aqui, acredito que é melhor usarmos o Button do próprio chakra, o que acha?

Comment (benits) [reply src/components/color-mode/utils.tsx:11]: esse é apenas para rodar os testes

**Pull request merged**